client/src/components/MeetingExperience.test.tsx

```diff
@@ -187,10 +187,48 @@ describe('MeetingExperience', () => {
 
     meetingExperience.update();
 
-    const survey = meetingExperience.find(Survey);
+    expect(setRenderPostCallMock).toBeCalled();
     const callWithChatComposites = meetingExperience.find(CallWithChatComposite);
-    expect(callWithChatComposites.length).toBe(0);
+    expect(callWithChatComposites.length).toBe(1);
+    const parentDiv = callWithChatComposites.parent();
+    expect(parentDiv.props().style.display).toBe('none');
+
+    const survey = meetingExperience.find(Survey);
     expect(survey.length).toBe(1);
-    expect(callWithChatComposites.length).toBe(0);
+  });
+
+  it('should not render Survey component', async () => {
```

Comment:
nit: Under what condition does this apply?

Same with the test case before, I think we should add more context to the test names:
- should render Survey component if postCall is defined
- should not render Survey component if postCall is undefined

Reply:
Fixed this